userspace/falco/app_actions/load_rules_files.cpp

```diff
@@ -162,16 +162,6 @@ application::run_result application::load_rules_files()
 		m_state->engine->enable_rule_by_tag(m_options.enabled_rule_tags, true);
 	}
 
-	/* Reading a scap file we have no concepts of ignored events we read all we need. */
-	if(!m_options.all_events && !is_capture_mode())
```

Comment:
It appeared I had to change the order of app_actions, broke old behavior of all_events option check. Re-added this one for example at a different place -> will be properly refactored with planned new `-A` behavior.